packages/strapi-plugin-content-manager/admin/src/containers/EditPage/saga.js

```diff
@@ -35,19 +36,29 @@ function* dataGet(action) {
   try {
     const modelName = yield select(makeSelectModelName());
     const params = { source: action.source };
-    const response = yield call(
-      request,
-      `/content-manager/explorer/${modelName}/${action.id}`,
-      { method: 'GET', params },
-    );
-
-    const pluginHeaderTitle = yield call(templateObject, { mainField: action.mainField }, response);
-    yield put(getDataSucceeded(action.id, response, pluginHeaderTitle.mainField));
+    const response = yield [
+      call(request, `/content-manager/explorer/${modelName}/${action.id}`, { method: 'GET', params }),
+      call(request, '/content-manager/layout', { method: 'GET', params }),
+    ];
+
+    const pluginHeaderTitle = yield call(templateObject, { mainField: action.mainField }, response[0]);
+    yield put(getDataSucceeded(action.id, response[0], pluginHeaderTitle.mainField));
+    yield put(getLayoutSucceeded(response[1]));
```

Comment:
Just as comment:

It's possible to use `Destructuring` to have more readable variables for response[0] and response[1]

`const [explorer, layout] = yield ...`